Assignment2/rlocre4-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,35 @@
+{
+	"@context":{
+		"student":"http://upm.com/student/",
+		"fullName":"student:fullName",
+		"hometown":"student:hometown",
+		"age":"student:age",
+		"almaMater":"student:almaMater",
+		"parentsName":"student:parentsName",
+		"subjectsSet":"student:subjectsSet",
+		"father":"parentsName:father",
+		"mother":"parentsName:mother",
+		"subjectsName":"subjectsSet:subjectsName",
+		"subjectsTeachers":"subjectsSet:subjectsTeachers",
+		"subjectsDescription":"subjectsSet:subjectsDescription"

```

Comment:
These attribute definitions work, but semantically they aren't ok. The best way to define them might be defining a new prefix for the subject concept and then use it to define their attributes. Same for father and mother.
You don't need to change them, but remember to provide good definitions for the future.